modules/custom/az_paragraphs/az_paragraphs_text_media/az_paragraphs_text_media.install

```diff
@@ -5,6 +5,8 @@
  * Install, update and uninstall functions for az_paragraphs_text_media module.
  */
 
+use Drupal\paragraphs\Entity\Paragraph;
```

Comment:
Was this important?

Reply:
No.  It was unrelated to the other changes in this PR so I omitted the change.